Intersect.Client/Entities/Player.cs

```diff
@@ -901,7 +877,7 @@ public void AutoTarget()
                     if (en.Value.GetEntityType() == EntityTypes.GlobalEntity ||
                         en.Value.GetEntityType() == EntityTypes.Player)
                     {
-                        if (en.Value != Globals.Me && !Globals.Me.IsInMyParty(en.Value))
+                        if (en.Value != Globals.Me || en.Value is Player player && Globals.Me.IsInMyParty(player))
```

Comment:
Also my bad, should be `if (en.Value != Globals.Me && !(en.Value is Player player && Globals.Me.IsInMyParty(player)))`.